tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -1345,6 +1345,29 @@ public function theUserUploadsAFileToSpace(
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" uploads a file inside federated space "([^"]*)" with content "([^"]*)" to "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $spaceName
+	 * @param string $content
+	 * @param string $destination
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theUserUploadsAFileToFedratedSpace(
+		string $user,
+		string $spaceName,
+		string $content,
+		string $destination
+	): void {
+		$spaceId = $this->featureContext->spacesContext->getSharesRemoteItemId($user, $destination);
+		$response = $this->featureContext->uploadFileWithContent($user, $content, $destination, $spaceId);
+		$this->featureContext->setResponse($response);
```

Comment:
we may could this too, bt not compulsary
```suggestion
		$this->featureContext->setResponse($this->featureContext->uploadFileWithContent($user, $content, $destination, $spaceId));
```

Reply:
the existing one is cleaner. let's try to split if the line gets more detailed

Reply:
or better formatting would do